src/platform/site-wide/mega-menu/tests/my-va-link/mega-menu.my-va.cypress.spec.js

```diff
@@ -65,7 +64,7 @@ const mockUser = {
   meta: { errors: null },
 };
 
-describe(manifest.appName, () => {
```

Comment:
just curious, since i've seen mention of the app ID or name from the manifest being used for things like running isolated unit tests or whatnots.  Is the reason to change this here in order to avoid importing from outside of platform directory?

Reply:
I'd say using data from the manifest could be useful for running tests in your application test folders, since this did do cross importing from one app to another, that would be something to avoid, so I hard coded it here. Also since the manifest.appName is 'dashboard' instead of My VA which is what we try and use instead I figured it would be better to just use that.

Reply:
gotcha, thanks! so is "dashboard" a legacy naming convention? may have been before my time here :)

Reply:
yea it was before my time as well. 

I think it was the original name when the application was first set up, and its really difficult to change the application name in the manifest once it has been set up / launched in prod so I think it has just stuck around.